audience.py

```diff
@@ -1,12 +1,16 @@
 import asyncio
+from logging import getLogger
 
 
 class Audience:
-    def __init__(self, robot):
+    def __init__(self, robot, log_level):
         """
         Audience keeps track of how many people have their hands up, and raises
         and lowers the robot's hand to match. This class is thread safe.
         """
+        self.logger = getLogger(__name__)
```

Comment:
Mild preference to rename `self._logger`, so you imply that this is not a public part of the class.